manifests/admin/config.pp

```diff
@@ -0,0 +1,14 @@
+# Pulp Admin Configuration
+class pulp::admin::config {
+  file { '/etc/pulp/admin/admin.conf':
+    ensure  => 'file',
+    content => template('pulp/admin.conf.erb'),
+  }
+
+  if $pulp::admin::puppet {
```

Comment:
Should I check based on $version = absent/purged and remove the files in this case? From my point of view the file should remain on the system even if the package is uninstalled.